src/test/resources/features/projectsWithJsonFile.feature

```diff
@@ -1,4 +1,11 @@
 Feature: Projects
+  Scenario: POST Project
+    When I send a "POST" request to "/projects" with json file "json/requestBodyProject.json"
+    And I save the response as "P"
+    Then I validate the response has status code 200
+    And I validate the response contains "name" equals "Project created by cucumber"
```

Comment:
Don't forget that projects has several attributes.
You should validate some of them.